pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -1395,5 +1736,21 @@ func (a *ApplicationRestoreController) cleanupResources(restore *storkapi.Applic
 			logrus.Errorf("unable to cleanup post restore resources, err: %v", err)
 		}
 	}
+	// Directly calling DeleteResourceExport with out checking backuplocation type.
+	// For other backuplocation type, expecting Notfound
+	crName := getResourceExportCRName(utils.PrefixRestore, string(restore.UID), restore.Namespace)
+	err := kdmpShedOps.Instance().DeleteResourceExport(crName, a.restoreAdminNamespace)
+	if err != nil && !k8s_errors.IsNotFound(err) {
+		errMsg := fmt.Sprintf("failed to delete resource export CR [%v]: %v", crName, err)
+		log.ApplicationRestoreLog(restore).Errorf("%v", errMsg)
+		return err
+	}
+	crName = getResourceExportCRName(utils.PrefixNFSRestorePVC, string(restore.UID), restore.Namespace)
+	err = kdmpShedOps.Instance().DeleteResourceExport(crName, restore.Namespace)
+	if err != nil && !k8s_errors.IsNotFound(err) {
+		errMsg := fmt.Sprintf("failed to delete resource export CR [%v]: %v", crName, err)
```

Comment:
It would good, if we some difference in the error message between these RE CR deletion in the errmsg